### DISCUSSION TITLE
Add NTP (time synchronization) to existing installation

### DISCUSSION THREAD

**Discussion by elli75** [RaspAP/raspap-webgui#1614] [Q&A]
Hello, I have a running installation of RaspAP and would now like to add time synchronization. Can someone help me with how to do this? I run RaspAP on a Raspberry PI 3B+ with RTC module and without an internet connection.
Thanks

**Comment by billz** [comment]
Hi, broadly speaking you have two options: 1) install ntp and configure it yourself as [described here](https://github.com/RaspAP/raspap-webgui/discussions/1553#discussioncomment-8911358), or 2) use the NTP management UI that is available as an [Insiders feature](https://docs.raspap.com/time/). If you'd like to evaluate the latter, let me know.
